packages/gatsby-kontent-components/src/image-element/index.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { GatsbyImageProps, GatsbyImage } from 'gatsby-plugin-image';
+import {
+  getGatsbyImageData,
+  GetGatsbyImageDataProps,
+} from './get-gatsby-image-data';
+
+export interface ImageItem {
+  image_id: string;
+  url: string;
+  description?: string;
+  name?: string;
+  height: number;
```

Comment:
Are width and height required to be set? I guess you could place the SVG.

Minimal would look like that:
```json
{
"image_id": "4452620d-6e5f-4446-b478-dc7a8297b6d3",
"description": null,
"url": "https://preview-assets-eu-01.kc-usercontent.com:443/75653ec1-36a2-01e0-0d25-a64799947697/443649a7-dd06-4861-97d5-4c96bfcb40c3/kk-logo-vert-color-neg-rgb.svg"
}
```

Reply:
I don't think this plugin is designed to work with SVGs. There is no mention of SVGs in the [official plugin documentation](https://www.gatsbyjs.com/plugins/gatsby-plugin-image/).